cli/cmd/runtime/start.go

```diff
@@ -126,7 +126,7 @@ func StartCmd(cliCfg *config.Config) *cobra.Command {
 			case "", "noop":
 				sink = activity.NewNoopSink()
 			case "kafka":
-				sink, err = activity.NewKafkaSink(conf.ActivitySinkKafkaBrokers, conf.ActivitySinkKafkaTopic)
+				sink, err = activity.NewKafkaSink(conf.ActivitySinkKafkaBrokers, conf.ActivitySinkKafkaTopic, *logger)
```

Comment:
The Zap logger is supposed to be passed as `*zap.Logger` (that's how it's returned from the library). So should update `KafkaSink` to take a `*zap.Logger` instead.